.ci/jobs/update-json-schema.yml

```diff
@@ -13,7 +13,7 @@
           discover-pr-forks-trust: permission
           discover-pr-origin: merge-current
           discover-tags: false
-          head-filter-regex: '^(?!update-stack-version).*$'
+          head-filter-regex: '^(master|PR-.*)$'
```

Comment:
PR automation should only be caring about changes coming from the master branch. Otherwise it will cause duplicated PRs with the json schema.